docs/data/data-grid/cells/cells.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Data Grid - Cells
+---
+
+# Data Grid - Cells
```

Comment:
I don't think we need a page for cells. Although it discusses cells, in reality it refers to the cells of a specific column. In this case, I would rename it something like "Column content" or move some sections from here to https://deploy-preview-4600--material-ui-x.netlify.app/x/react-data-grid/column-definition/. To make the page shorter I think we can remove some sections. If we move it back to Column definition I propose the following structure:

- Column definition
  - Providing content
    1. Value getter
    2. Value setter (remove, only used for editing)
    3. Value formatter
    4. Value parser (maybe remove, is used for both filtering and editing, but editing already explains extensively, filtering could duplicate the content)
  - Rendering content
    - Custom cells
    - Making cells accessible (could link to https://github.com/mui/mui-x/pull/4254)
    - Render edit cell (remove)

Reply:
I applied a structure similar to what you proposed